iOSClient/Viewer/NCViewerMedia/NCViewerMediaPage.swift

```diff
@@ -117,11 +116,11 @@ class NCViewerMediaPage: UIViewController {
 
         metadatas.removeAll()
```

Comment:
@marinofaggiana fixed it and also a potential memory leak..
 I'm not sure why you clear the array here. After the view disappears / is reinitialised it shouldn't matter, should it?
Also, I don't think we need to remove all the observers that use selectors. wdyt?

Reply:
@thisIsTheFoxe  theoretically not, but many recommend it ... so it doesn't hurt, but the refresh after the modify still do not works. Do you have try it ?